### DISCUSSION TITLE
Line chart single value issue

### DISCUSSION THREAD

**Discussion by GeorgiyShur** [patrykandpatrick/vico#905] [Questions] [answered]
### Question

When my line chart has just single value in the series, it is displayed in the start of the chart, no labels are displayed on the X-axis and also the custom persistent marker doesn't get the color from custom line fill.

Is it possible to display single value as a point (possibly with persistent marker), centered horizontally with a value on the X-axis?

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @GeorgiyShur. To clarify, do you have a `PointProvider` applied?

**Reply by GeorgiyShur** [reply]
No I don't... Tbh I don't see a class like this in the sources for this version of Vico (`2.0.0-alpha.28`)

**Reply by Gowsky** [reply, marked as answer]
@GeorgiyShur, it's in `LineCartesianLayer`, and this is what you should use to display a point for the single entry. See the `pointProvider` parameter of `rememberLine`. You can make the `PointProvider` conditional, so no points are displayed when there are multiple entries.

Once you've done this, and once #910 has been fixed, your persistent marker will work as expected. (See [here](https://github.com/patrykandpatrick/vico/issues/910#issuecomment-2395121839) for a temporary solution.)

Moving on to centering the point horizontally and and getting the _x_-axis to display a label for it, this should happen automatically. We've confirmed this in the sample app. If you're seeing different behavior, please share a minimal reproducible example that demonstrates it.

**Reply by GeorgiyShur** [reply]
Ok got it, thanks a lot!
